client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -178,7 +178,7 @@ export function getEnhancedTasks(
 					taskData = {
 						title: translate( 'Write your first post' ),
 						completed: firstPostPublishedCompleted,
-						disabled: mustVerifyEmailBeforePosting || false,
+						disabled: mustVerifyEmailBeforePosting || isWriteFlow( flow || null ) || false,
```

Comment:
I think we want to use `isStartWritingFlow` rather than commandeering `isWriteFlow` since it's being used in a couple places.

Reply:
Good catch @DustyReagan, [changed it here](https://github.com/Automattic/wp-calypso/pull/76249/commits/e85b55b40e614cc8a44459f8782c4aa8ab3bdb07).